photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipeline.java

```diff
@@ -163,6 +187,8 @@ protected CVPipelineResult process(Frame frame, AprilTagPipelineSettings setting
             targetList.add(target);
         }
 
+        croppedFrame.output.release();
```

Comment:
So if fullyCovers up in croppipe is true, this mat is a straight copy of the input mat. We cannot release this mat early in that case. This will probably break video streaming. Only release it if the mat is actually a submat of the input image.